code-samples.json

```diff
@@ -0,0 +1,23 @@
+{
+    "actors-sequential.pony": {
+        "runnable": true,
+        "stdout": "This is printed first\nThis is printed last\n",
+        "stderr": null,
+        "exitcode": 0
+    },
+    "traits-and-interfaces-private-methods.pony": {
+        "runnable": true,
+        "stdout": null,
+        "stderr": "interfaces can't have private methods, only traits can\n",
+        "exitcode": 1
+    },
+    "actors-behaviors.pony": {
+        "runnable": false
+    },
+    "appendices-examples-access-command-line-arguments.pony": {
+        "runnable": true,
+        "stdout": "This is printed first\nThis is printed last\n",
```

Comment:
We shouldn't need to test `stdout` of running the examples - we just need to prove that an example successfully compiles (or fails to compile with a particular message in `stderr`). We don't need to run the examples at all - just compile them.

Reply:
What's the point of a snippet if it does something but we don't even know if it illustrates what the chapter talks about? It just happens to have a few buzzwords in it.